pkgs/development/python-modules/cairosvg/default.nix

```diff
@@ -1,47 +1,52 @@
 {
   lib,
   buildPythonPackage,
+  fetchFromGitHub,
+
+  # build-system
+  setuptools,
+
+  # dependencies
   cairocffi,
   cssselect2,
   defusedxml,
-  fetchPypi,
   pillow,
-  pytestCheckHook,
-  setuptools,
   tinycss2,
+
+  # testing
+  pytestCheckHook,
 }:
 
 buildPythonPackage rec {
   pname = "cairosvg";
-  version = "2.7.1";
+  version = "2.8.2";
   pyproject = true;
 
-  src = fetchPypi {
-    pname = "CairoSVG";
-    inherit version;
-    hash = "sha256-QyUx1yNHKRuanr+2d3AmtgdWP9hxnEbudC2wrvcnG6A=";
+  src = fetchFromGitHub {
+    owner = "Kozea";
+    repo = "CairoSVG";
+    tag = version;
+    hash = "sha256-KWUZA8pcHMnDEkAYZt3zDzPNynhGBuLZuagNPfHF8EA=";
   };
 
-  nativeBuildInputs = [ setuptools ];
+  build-system = [ setuptools ];
 
-  propagatedBuildInputs = [
+  dependencies = [
     cairocffi
     cssselect2
     defusedxml
     pillow
     tinycss2
   ];
 
-  propagatedNativeBuildInputs = [ cairocffi ];
+  nativeBuildInputs = [ cairocffi ];
 
-  nativeCheckInputs = [ pytestCheckHook ];
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
```

Comment:
Spurious format change in the update commit, I'd at least move it back to the previous commit.

Reply:
Fixed.